lib/graphql/generators/templates/schema.erb

```diff
@@ -0,0 +1,27 @@
+<%= schema_name %> = GraphQL::Schema.define do
+  query(Types::QueryType)
+<% if options[:relay] %>
+  # Relay Object Identification:
+
+  # Return a string UUID for `object`
+  id_from_object ->(object, type_definition, query_ctx) {
+    # Here's a simple implementation which:
+    # - joins the type name & object.id
+    # - encodes it with base64:
+    # GraphQL::Schema::UniqueWithinType.encode(type_definition.name, object.id)
+  }
+
+  # Given a string UUID, find the object
+  object_from_id ->(id, query_ctx) {
+    # For example, to decode the UUIDs generated above:
+    # type_name, item_id = GraphQL::Schema::UniqueWithinType.decode(id)
+    #
+    # Then, based on `type_name` and `id`
+    # find an object in your application
+    # ...
+  }
+<% end %><% if options[:batch] %>
+  # GraphQL::Batch setup:
+  lazy_resolve(Promise, :sync)
+  instrument(:query, GraphQL::Batch::Setup)
```

Comment:
Somewhat off-topic, but I wonder if we could turn this into a single line, say:

```ruby
GraphQL::Schema.define do
  # ...
  use_graphql_batch
end
```

And within `graphql-batch` we could do:

```ruby
GraphQL::Schema.accepts_definitions(
  use_graphql_batch: ->(schema) {
    return if schema.metadata[:graphql_batch_setup]
    schema.lazy_resolve(Promise, :sync)
    schema.instrument(:query, GraphQL::Batch::Setup)
    schema.metadata[:graphql_batch_setup]
  }
)
```

cc @dylanahsmith